app/src/main/res/layout/recent_patients.xml

```diff
@@ -7,7 +7,8 @@
   <androidx.recyclerview.widget.RecyclerView
     android:id="@+id/recent_patient_recyclerview"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content" />
+    android:layout_height="wrap_content"
+    android:layout_marginBottom="@dimen/spacing_96" />
```

Comment:
Btw, did you intentionally capitalize `Recent` in this commit message? Looks… odd one out

`Add bottom margin to Recent patient recycler view`

Reply:
Yes I did.

Reply:
oh okay